lib/workers/branch/execute-post-upgrade-commands.ts

```diff
@@ -0,0 +1,178 @@
+import is from '@sindresorhus/is';
+import minimatch from 'minimatch';
+import { getAdminConfig } from '../../config/admin';
+import { addMeta, logger } from '../../logger';
+import { exec } from '../../util/exec';
+import { readLocalFile, writeLocalFile } from '../../util/fs';
+import { File, getRepoStatus } from '../../util/git';
+import { regEx } from '../../util/regex';
+import { sanitize } from '../../util/sanitize';
+import * as template from '../../util/template';
+import { BranchConfig, BranchUpgradeConfig } from '../types';
```

Comment:
```suggestion
import type { BranchConfig, BranchUpgradeConfig } from '../types';
```

type only import, will be removed at runtime

Reply:
Done